doc/source/release-notes/Percona-Server-8.0.26-16.rst

```diff
@@ -0,0 +1,44 @@
+.. _8.0.26-16:
+
+================================================================================
+*Percona Server for MySQL* 8.0.26-16
+================================================================================
+
+:Date: October 6, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.26-16
+includes all the features and bug fixes available in the
+`MySQL 8.0.26 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-24.html>`__.
+in addition to enterprise-grade features developed by Percona.
+
+
+Deprecated Features
+=================================================
+
+The TokuDB Storage Engine is disabled in Percona Server 8.0.26-16.
```

Comment:
Can we please elaborate on what this means in more detail? Is there a section in the documentation that we can point to here that explains the behavior that TokuDB users will experience and how they can temporary re-enable the engine to perform the migration? Since this is a disruptive change that may be surprising, we need to take extra care of guiding our users here. The blog post below gives some hints, but this deserves being mentioned more explicitly.

Reply:
I've updated each TokuDB topic with a header with deprecation information. I rewrote this section to point to those pages.